TACC/TACC-Docs
# fix: delete test pages (so not shown in search)

## Overview

Delete test pages from search, because #111 does not work on [prod site](https::/docs.tacc.utexas.edu).

## Related

- [TUP-458](https://jira.tacc.utexas.edu/browse/TUP-458)
- [Post Launch Docs Meeting](https://docs.google.com/document/d/1cnq4IIb4jI4AFW8o0bPgkBnjRmFrrCkHn8G2NtNzrVM/edit)

## Changes

- **removed** pages that should not appear in search

## Testing

0. Prepare code to test:
    ```
    # checkout commit before search exclusion pr #111
    git checkout a0a4a03673feae36ef4d1c45eaa170fea29d2ee5
    # make new temporary branch
    git checkout -b test/remove-test-page-results-from-search
    # merge this branch on/into it
    git merge bugfix/remove-test-page-results-from-search
    ```
1. `docker compose build`
2. `docker compose up`
3. Search for "lmod".
4. Verify results do **not** have:
    > "L" to the "M" to the "O" […]

## UI

| before | after |
| - | - |
| ![before](https://user-images.githubusercontent.com/62723358/233453229-1a8ca33b-cae0-44c9-83c0-80c0e6b1cfd8.png) | ![after](https://user-images.githubusercontent.com/62723358/233453232-4feab563-057b-484c-9a7c-eac01f6880e0.png) |
